llm4pddl/approaches/llm_open_loop_approach.py

```diff
@@ -108,13 +112,19 @@ def _create_prompt_prefix(self, dataset: Dataset) -> None:
         prompts = []
         prompt_dataset = dataset[:FLAGS.num_prompt_tasks]
         for datum in prompt_dataset:
-            prompt = self._create_prompt(datum.task, datum.solution)
+            if use_random_object_names:
+                prompt = self._create_prompt(datum.task, datum.solution,
+                                             self._rng)
+            else:
+                prompt = self._create_prompt(datum.task, datum.solution)
             prompts.append(prompt)
         self._prompt_prefix = "\n\n".join(prompts) + "\n\n"
         logging.debug(f"Created prompt prefix:\n{self._prompt_prefix}")
 
     @staticmethod
-    def _create_prompt(task: Task, plan: Optional[Plan] = None) -> str:
+    def _create_prompt(task: Task,
```

Comment:
looks like we're not randomizing the evaluation task. should we? probably yes. so we need to invert the substitution in solve() too.